docs/usage/docker.md

```diff
@@ -117,6 +117,26 @@ If you wish to enable major versions then add the preset `docker:enableMajor` to
 Renovate has some Docker-specific intelligence when it comes to versions.
 For example:
 
+### Ubuntu codenames
+
+Renovate understands [Ubuntu release codename](https://wiki.ubuntu.com/Releases) tags and will offer upgrading them to the latest LTS release available (e.g. from `ubuntu:xenial` to `ubuntu:focal`) as long as it is inline with the following naming scheme:
```

Comment:
```suggestion
Renovate understands [Ubuntu release codename](https://wiki.ubuntu.com/Releases) tags and will offer upgrading them to the latest LTS release available (e.g. from `ubuntu:xenial` to `ubuntu:focal`).
For this to work you must follow this naming scheme:
```

Split up the sentence into smaller parts, and rewrite the platform for the list.